tests/foreman/ui/test_subscription.py

```diff
@@ -210,10 +210,13 @@ def test_positive_access_with_non_admin_user_with_manifest(
         default_organization=org,
     ).create()
     with target_sat.ui_session(test_name, user=user.login, password=user_password) as session:
-        assert (
-            session.subscription.search(f'name = "{DEFAULT_SUBSCRIPTION_NAME}"')[0]['Name']
-            == DEFAULT_SUBSCRIPTION_NAME
-        )
+        all_subscriptions = session.subscription.search('name ~ "Red Hat"')
+        for subscription in all_subscriptions:
+            if subscription['Name'] == DEFAULT_SUBSCRIPTION_NAME:
+                assert True
+                break
+        else:
+            pytest.fail(f"{DEFAULT_SUBSCRIPTION_NAME} not found..!")
```

Comment:
This should do the same I think (haven't tried locally though)
```suggestion
        assert any(sub['Name'] == DEFAULT_SUBSCRIPTION_NAME for sub in all_subscriptions)
```

Btw, why didn't the original search work? I'm guessing it should return just one result (if the subscription is present).

Reply:
Original search doesn't work, I changed UI selector and tried, but again when we are on page, view not able to click on Search button.